frontend/apps/web/src/components/system/Overview.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import { Model } from '@/model'
+import {
+  CardLayout,
+  ProgressBar,
+  StackLayout,
+  Text,
+  Table,
+  Button,
+} from '@filecoin/ui'
+import { useNavigate } from 'react-router-dom'
+import styled from 'styled-components'
+
+const DATASCORE = {
+  good: 'success',
+  bad: 'error',
+  mediocre: 'warning',
+}
+
+interface Props {
+  model: Model
+  modelName: string
+}
+
+export const Overview: React.FC<Props> = ({ model, modelName }) => {
+  const navigate = useNavigate()
+  const system = model.findSystemByName(modelName)
+  const totalSubsystems = system.subsystems.length
+
+  return (
+    <Wrapper>
+      <ProgressBarWrapper>
+        <ProgressBar
+          data={system.testKindStats.percentages.map(
+            ({ kind, percentage }) => ({
+              name: kind,
+              percentage,
+            }),
+          )}
+          onClick={() => null}
+          legend
+        ></ProgressBar>
+      </ProgressBarWrapper>
+      <ProgressBarWrapper>
+        <ProgressBar
+          data={system.testStatusStats.percentages.map(
+            ({ status, percentage }) => ({
+              name: status,
+              percentage: percentage,
+            }),
+          )}
+          onClick={() => null}
+          legend
+        ></ProgressBar>
+      </ProgressBarWrapper>
+      <TableWrapper>
+        <Text type={'subtitle l'}>Subsystems ({totalSubsystems})</Text>
+        <TableStyled
+          variant="default"
+          columns={{
+            Subsystems: {
+              header: 'Subsystem',
+              width: 150,
+              Cell: ({ data }) => <Text type="text s">{data.name}</Text>,
+            },
+            testKinds: {
+              header: 'Test Kinds',
+              width: 225,
+              Cell: ({ data }) => (
+                <ProgressBar
+                  onClick={() => navigate('/repository-details')}
```

Comment:
remove all redirecting here, since we don't have a link where to go

Reply:
done